.version

```diff
@@ -1 +1 @@
-0.1.3
+0.2.0
```

Comment:
Decided to merge bps/nats -> bps/stan rename into this branch - in the end, let's have one bigger release.